src/index.js

```diff
@@ -1,60 +1,55 @@
+/* eslint-disable default-case */
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable **all** linter checks on your code to ensure that your code is robust code and clean.

Reply:
- [ ] As requested by the previous reviewer, Please enable all linter checks on your code to ensure that your code is robust code and clean.